README.md

```diff
@@ -19,6 +19,24 @@ This package provides Kolibri users with the ability to authenticate against an
 3. Restart Kolibri
 
 
+## Used claims
+This plugin will create a new user in the Kolibri database after it authenticates using the OIDC provider.
+From the [standard OIDC claims](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims) the plugin will fetch these ones to add the information to the database:
+
+- `nickname` (or `username`)
+- `given_name`
+- `family_name`
+- `email`
+- `birthdate`
+- `gender`
+
+Apart from these standard claims, the plugin will accept a list of `roles` in the user_info token provided in a string with the format "roles":[role1, role2....]
+As an user_info token payload example:
+```json
+{"email":"jhon@doe.com", "username":"jdoe", "roles":["coach","admin"], "family_name":"Doe"}
+```
```

Comment:
We probably want to specify that if no role is provided, `learner` is assumed.

Also from a user's perspective in Kolibri, there is only one role at a time - not a list of roles. Therefore I think this should probably take just a single role, e.g. 

```json
{
  "email": "jhon@doe.com",
  "username": "jdoe",
  "role": "coach",
  "family_name": "Doe"
}
```

Reply:
mmm, in Kolibri you can be added to the coach and admin roles at the same time. Code allows it. Not sure if we should limit it in this plugin.

Reply:
I believe the main reason we allow multiple roles is so that [they can be to specific collections](https://kolibri-dev.readthedocs.io/en/develop/backend_architecture/uap/concepts_and_definitions.html#roles). For example, a user might have an admin role on the facility collection, and a coach role on a class collection.

There isn't currently any meaningful interpretation of a user having multiple roles on the same collection (here, the facility), and I'm not sure how the UI would handle that situation.

Reply:
I have kept the roles as a list because that's how OIDC providers return them, even if there's only one